poll/public/handlebars/poll_studio.handlebars

```diff
@@ -2,21 +2,26 @@
 {{#each items}}
 <li class="field comp-setting-entry is-set poll-{{noun}}-studio-item">
     <div class="wrapper-comp-setting">
+        <div class="poll-move">
+            <button class="poll-move-up">&#9650;<span class="sr">&nbsp;move poll up</span></button>
+            <button class="poll-move-down">&#9660;<span class="sr">&nbsp;move poll down</span></button>
+        </div>
+        <button class="button action-button poll-delete-answer">Delete</button>
         <label class="label setting-label poll-setting-label" for="{{noun}}-label-{{key}}">{{noun}}</label>
         <input class="input setting-input" name="{{noun}}-label-{{key}}" id="{{noun}}-label-{{key}}" value="{{text}}" type="text" /><br />
         {{#if image}}
         <label class="label setting-label" for="{{noun}}-img-{{key}}">Image URL</label>
-        <input class="input setting-input" name="{{noun}}-img-{{key}}" id="{{noun}}-img-{{key}}" value="{{img}}" type="text" />
+        <input class="input setting-input" name="{{noun}}-img-{{key}}" id="{{noun}}-img-{{key}}" value="{{img}}" type="text" /><br />
+        <label class="label setting-label" for="{{noun}}-img_alt-{{key}}">Image alternate text</label>
+        <input class="input setting-input" name="{{noun}}-img_alt-{{key}}" id="{{noun}}-img_alt-{{key}}" value="{{img_alt}}" type="text" /><br />
```

Comment:
I don't understand how making this field required would cause _any_ issues with existing units; it's a Studio-side validation. Only new units, or old units that are re-edited, would face the requirement. Existing, published units would be totally fine, right?